docs/layouts/shortcodes/generated/checkpointing_configuration.html

```diff
@@ -44,6 +44,42 @@
             <td>String</td>
             <td>The default directory used for storing the data files and meta data of checkpoints in a Flink supported filesystem. The storage path must be accessible from all participating processes/nodes(i.e. all TaskManagers and JobManagers). If the 'state.checkpoint-storage' is set to 'jobmanager', only the meta data of checkpoints will be stored in this directory.</td>
         </tr>
+        <tr>
+            <td><h5>state.checkpoints.file-merging.across-checkpoint-boundary</h5></td>
+            <td style="word-wrap: break-word;">false</td>
+            <td>Boolean</td>
+            <td>Only relevant if <code class="highlighter-rouge">state.checkpoints.file-merging.enabled</code> is enabled.<br />Whether to allow merging data of multiple checkpoints into one physical file. If this option is set to false, only merge files within checkpoint boundaries will be merged. Otherwise, it is possible for the logical files of different checkpoints to share the same physical file.</td>
+        </tr>
+        <tr>
+            <td><h5>state.checkpoints.file-merging.enabled</h5></td>
```

Comment:
Is it possible adjust the order of each document item/entry? This one better be the first.

Reply:
I'm afraid not, it's in alphabetical order. 
For example, the configuration options related to `speculative ` are also like this.
![image](https://github.com/apache/flink/assets/18653940/117a8695-3733-4454-8f58-70d3ea0c9f9e)

Reply:
I found the annotation `@Documentation.Section` can specify `position`, may be helpful?

Reply:
The `position` can only determine the order of `xxx_secttion.html`.  `checkpointing_configuration.html` is generated by `ConfigOptionsDocGenerator#generateTablesForClass`, it used `DocumentedKey` as the [comparing key](https://github.com/apache/flink/blob/master/flink-docs/src/main/java/org/apache/flink/docs/configuration/ConfigOptionsDocGenerator.java#L547).

Reply:
I see. Thus we leave it be.